src/luxonis_ml/data/dataset.py

```diff
@@ -397,162 +241,97 @@ def _init_path(self) -> None:
                 / "data"
                 / self.team_id
                 / "datasets"
-                / self.dataset_id
+                / self.dataset_name
             )
             os.makedirs(self.path, exist_ok=True)
+            self.media_path = os.path.join(self.path, "media")
+            self.annotations_path = os.path.join(self.path, "annotations")
+            self.metadata_path = os.path.join(self.path, "metadata")
+            os.makedirs(self.media_path, exist_ok=True)
+            os.makedirs(self.annotations_path, exist_ok=True)
+            os.makedirs(self.metadata_path, exist_ok=True)
         elif self.bucket_storage.value == "s3":
```

Comment:
Given the path is the same should we just set the prefix to "GCS" and "S3" instead of having both?

Reply:
Instead of them being separate*

Reply:
Fixed